src/devices/Pigpio/PigpiodIf.cs

```diff
@@ -0,0 +1,2789 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+// Disable all XML Comment warnings in this file as copied from elsewhere //
```

Comment:
Is this copied from https://github.com/Rapidnack/PigpiodIfTest ?

Reply:
Yes, it looks like a wrapper around [Socket Interface](http://abyz.me.uk/rpi/pigpio/sif.html) which is unencrypted.